benchmark_runner/common/grafana/grafana_operations.py

```diff
@@ -0,0 +1,102 @@
+
+import requests
+import json
+
+
+class GrafanaOperations:
+    """
+    This class responsible for Grafana operations
+    """
+    def __init__(self, grafana_url: str, grafana_api_key: str, grafana_json: str):
+        self.grafana_url = grafana_url
+        self.grafana_api_key = grafana_api_key
+        self.grafana_json = grafana_json
+        self.dashboard_data = {}
+
+    def fetch_all_dashboards(self):
+        """
+        This method fetches all dashboards
+        :return:
+        """
+        dashboard_list = []
+        headers = {
```

Comment:
Comment still applies.

Reply:
Shouldn't the http request have a Content-Type: header since you expect JSON?

Reply:
The Content-Type header is typically used when sending data in a request (e.g., in a POST or PUT request) to specify the format of the data being sent. Since I am only fetching data using a GET request and not sending any data, I don't need to include the Content-Type header in this case.
You can see that I use content type json={"dashboard": self.dashboard_data} in override_dashboard() below